### DISCUSSION TITLE
[Feature] Merge/Upload images from fallback datasource automaticy

### DISCUSSION THREAD

**Discussion by wafl93** [immich-app/immich#11478] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

My immich - folder is located on my OMV - Nas and is loaded into immich on boot.
It happened already a few times, that my NAS was not reachable so immich used a "fallback storage" where the images where uploaded to.
When i reboot immich and my storage is working again, the images are in the immich - GUI but the files cannot be loaded because they have been saved on the fallback storage.
I have no idea where this storage is located, so my solution was to delete these images in the GUI, removed them from trash and uploaded the files manually again via the Webinterface.
It would be great, when immich could automatically move all files from the "fallback storage" to the real storage as soon it becomes available again.

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
This is all handled on the OS filesystem mount level, Immich is not aware of the network share or whether it's mounted correctly. If it's not mounted and not correctly configured to error, the files are written to the root disk at the mount path. Making sure it's configured correctly is on you as the system administrator.
